members/us_sen_ga_kelly_loeffler.yaml

```diff
@@ -121,7 +121,7 @@ contact_form:
       - value: Submit
         selector: button#edit-actions-submit
     - find:
-      - selector: div.wrap-messages div
+      - selector: div.wrap-messages > div > h4
```

Comment:
html structure based selectors should really be avoided if possible since they extremely brittle. Would `div.wrap-messages div h4` work instead?

Reply:
It should work, tho now the success message is not appearing at all. I'll try again in a few

Reply:
I can confirm this works @tirish